script/Arcueid/Day 6/06_00_ARC06_1.txt

```diff
@@ -349,7 +349,7 @@ It's clear to see that my standing within the Tohno household has plummeted to t
 [sha:ed1f4f01b7add8a7190b238d6dd8e55a9207cc1b]{
 -- Page 26, Offset 13989.
 -- 屋敷の主は秋葉なんだから、秋葉が怒っているという事は、こっちに味方はいないのは当然と言える。
-Since Akiha is the head of the mansion, it's no surprise that I have no allies when she's upset.
+Since Akiha is the head of the mansion, it's no surprise I have no allies when she's upset.
```

Comment:
Might keep this `that`. There's a couple removed in this PR - personal preference?

Reply:
I agree with Irish on this one